src/shared/modules/itemEditor/itemEditorDuck.js

```diff
@@ -96,10 +114,11 @@ export const handleEditEntityEpic = (action$, store) =>
     let cmd
     switch (action.editType) {
       case 'create':
+        getCypherCompatibleValue(action)
         if (action.entityType === 'nodeProperty') {
           cmd = `MATCH (a)
         WHERE ID(a) = ${action.editPayload.id}
-        SET a.${action.editPayload.key} = '${action.editPayload.value}'
+        SET a.${action.editPayload.key} = ${convertedValue}
```

Comment:
call getCypherCompatibleValue() method to get value in here

Reply:
done